docs/medical-api/api-reference/fhir/consolidated-data-query-post.mdx

```diff
@@ -42,6 +42,12 @@ You can check the status of the data query by calling [get consolidated data que
   ISO 8601. If not provided, no end date filter will be applied.
 </ParamField>
 
+<ParamField query="conversionType" type="string">
+  The doc type to convert to. Either `html` or `pdf`.
+
+  <Info>This parameter is to indicate how the medical record should be rendered.</Info>
+</ParamField>
```

Comment:
We should probably be more thorough with the explanation here, as this is a pretty major modification to the workflow, isn't mentioned in the main description, and hidden away in this parameter.

ie we need to make it obvious that the dev can use this endpoint to make render a medical record summary from the patient's FHIR data, which will then be returned as an attachement in the DocumentReference url - would also suggest updating the webhook docs accordingly for an example with instructions on how to get the file.